ios/FluentUI/Badge Field/BadgeViewTokenSet.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Pre-defined styles of the Badge.
+@objc(MSFBadgeViewStyle)
+public enum Style: Int {
+    case `default`
+    case danger
+    case severeWarning
+    case warning
+    case success
+    case neutral
+}
+
+/// Pre-defined sizes of the Badge.
+@objc(MSFBadgeViewSize)
+public enum Size: Int, CaseIterable {
+    case small
+    case medium
+
+    var labelTextStyle: AliasTokens.TypographyTokens {
```

Comment:
I think this should be in the tokenSet.

Reply:
`AliasTokens.TypographyTokens` is not a `ControlTokenValue` so I'm not sure it's possible.